### ISSUE TITLE
'updateObject' now fails after updating packages

### ISSUE THREAD

**Issue by sjmonkley** [hbc/bcbioRNASeq#146]
I have successfully run the Differential Expression R Markdown for bcbioRNAseq many times using the bcb.rda object generated from the bcbio RNASeq pipeline. 
In the past this has required the updateObject() to be run in order to update the bcb object from 0.2.9 to 0.3.25.
After updating various packages this function now generates an error:

```
> object <- updateObject(object)# adapted from commented code in intro sectionobject
> Updating bcbioRNASeq object from version 0.2.9 to 0.3.25.
> Coercing 'programVersions' to DataFrame.
> Simplifying stashed 'sessionInfo'.
> Dropping legacy 'template'.
> Coercing 'tx2gene' to Tx2Gene class.
> Renaming 'length' assay to 'avgTxLength'.
> Moving 'rowRangesMetadata' into 'rowRanges' metadata.
> Error in validObject(.Object) : 
>   invalid class "bcbioRNASeq" object: Class checks failed:
> [1] "dataVersions"    "programVersions"
```
Session Info
```
> sessionInfo()
R version 3.6.1 (2019-07-05)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 18362)

Matrix products: default

locale:
[1] LC_COLLATE=Swedish_Sweden.1252  LC_CTYPE=Swedish_Sweden.1252    LC_MONETARY=Swedish_Sweden.1252 LC_NUMERIC=C                    LC_TIME=Swedish_Sweden.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] DESeq2_1.26.0               bcbioRNASeq_0.3.25          basejump_0.11.9             SummarizedExperiment_1.16.1 DelayedArray_0.12.2         BiocParallel_1.20.1        
 [7] matrixStats_0.55.0          Biobase_2.46.0              GenomicRanges_1.38.0        GenomeInfoDb_1.22.0         IRanges_2.20.2              S4Vectors_0.24.3           
[13] BiocGenerics_0.30.0        

loaded via a namespace (and not attached):
  [1] backports_1.1.5               Hmisc_4.3-0                   AnnotationHub_2.18.0          BiocFileCache_1.10.2          plyr_1.8.5                    lazyeval_0.2.2               
  [7] splines_3.6.1                 DESeqAnalysis_0.2.4           ggplot2_3.2.1                 digest_0.6.23                 ensembldb_2.10.2              htmltools_0.4.0              
 [13] fansi_0.4.1                   magrittr_1.5                  checkmate_1.9.4               memoise_1.1.0                 cluster_2.1.0                 syntactic_0.2.3              
 [19] limma_3.42.2                  transformer_0.2.4             Biostrings_2.54.0             annotate_1.64.0               R.utils_2.9.2                 askpass_1.1                  
 [25] prettyunits_1.1.1             jpeg_0.1-8.1                  colorspace_1.4-1              blob_1.2.1                    rappdirs_0.3.1                ggrepel_0.8.1                
 [31] xfun_0.12                     dplyr_0.8.4                   tximport_1.14.0               crayon_1.3.4                  RCurl_1.98-1.1                genefilter_1.68.0            
 [37] survival_3.1-8                glue_1.3.1                    gtable_0.3.0                  zlibbioc_1.32.0               XVector_0.26.0                UpSetR_1.4.0                 
 [43] SingleCellExperiment_1.8.0    scales_1.1.0                  pheatmap_1.0.12               edgeR_3.28.0                  DBI_1.1.0                     Rcpp_1.0.3                   
 [49] xtable_1.8-4                  progress_1.2.2                htmlTable_1.13.3              foreign_0.8-75                bit_1.1-15.1                  Formula_1.2-3                
 [55] htmlwidgets_1.5.1             httr_1.4.1                    RColorBrewer_1.1-2            acepack_1.4.1                 bioverbs_0.2.5                pkgconfig_2.0.3              
 [61] XML_3.99-0.3                  R.methodsS3_1.7.1             nnet_7.3-12                   dbplyr_1.4.2                  locfit_1.5-9.1                tidyselect_1.0.0             
 [67] rlang_0.4.4                   reshape2_1.4.3                later_1.0.0                   AnnotationDbi_1.48.0          munsell_0.5.0                 BiocVersion_3.10.1           
 [73] tools_3.6.1                   cli_2.0.1                     RSQLite_2.2.0                 ggridges_0.5.2                stringr_1.4.0                 fastmap_1.0.1                
 [79] yaml_2.2.1                    knitr_1.28                    bit64_0.9-7                   acidplots_0.2.13              bcbioBase_0.6.9               purrr_0.3.3                  
 [85] AnnotationFilter_1.10.0       mime_0.8                      R.oo_1.23.0                   grr_0.9.5                     biomaRt_2.42.0                brio_0.3.5                   
 [91] compiler_3.6.1                rstudioapi_0.10               curl_4.3                      png_0.1-7                     interactiveDisplayBase_1.24.0 tibble_2.1.3                 
 [97] geneplotter_1.64.0            stringi_1.4.4                 GenomicFeatures_1.38.1        lattice_0.20-38               ProtGenerics_1.18.0           Matrix_1.2-18                
[103] vctrs_0.2.2                   pillar_1.4.3                  lifecycle_0.1.0               BiocManager_1.30.10           data.table_1.12.8             cowplot_1.0.0                
[109] bitops_1.0-6                  Matrix.utils_0.9.7            httpuv_1.5.2                  rtracklayer_1.46.0            R6_2.4.1                      latticeExtra_0.6-29          
[115] promises_1.1.0                gridExtra_2.3                 sessioninfo_1.1.1             assertthat_0.2.1              freerange_0.2.3               openssl_1.4.1                
[121] withr_2.1.2                   GenomicAlignments_1.22.1      Rsamtools_2.2.1               GenomeInfoDbData_1.2.2        hms_0.5.3                     grid_3.6.1                   
[127] rpart_4.1-15                  DelayedMatrixStats_1.8.0      goalie_0.3.5                  shiny_1.4.0                   base64enc_0.1-3           
```

**Comment by mjsteinbaugh** [comment]
Thanks @sjmonkley, I'll take a look and push a point release that fixes this.

**Comment by mjsteinbaugh** [comment]
OK this should be fixed in v0.3.31. See related pull request https://github.com/hbc/bcbioRNASeq/pull/147. I'll merge this into master and create a release once the build checks pass.

Best,
Mike
